src/vm/node_modules/VM.js

```diff
@@ -8688,11 +8697,11 @@ function installZone(payload, log, callback)
             // Create any filesystems now that have the 'create' flag set.
             // Note that currently if you reprovision, all these created volumes
             // will be destroyed and recreated as well since they're children of
-            // the zoneroot. This only currently works with "docker" VMs.
+            // the zoneroot. This does not work with HVMs.
```

Comment:
nit: does not currently work with HVMs?

Reply:
I don’t want to set an expectation that someday it might. Even when/if p9fs comes about, I don’t know if this particular mechanism will be used with it. If it does someday/somehow change, I don’t think anybody will object because of this comment :-)